src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPart.SummaryAdmin.cshtml

```diff
@@ -0,0 +1,5 @@
+@model OrchardCore.ContentManagement.Display.ViewModels.ContentItemViewModel
+
+<a class="btn btn-sm text-secondary" data-toggle="tooltip" title="@T["List Items"]" admin-for="@Model.ContentItem">
+    <i class="fa fa-list"></i>
```

Comment:
just needs the merge conflicts resolving (just revert the new line you added) and the ui changing to remove the icons.